format/Schema.fbs

```diff
@@ -501,6 +501,24 @@ struct Buffer {
   length: long;
 }
 
+/// ----------------------------------------------------------------------
```

Comment:
Field node seems more a part of message than schema?  If it is needed as a top level type lets make it a separate file?